style/components/profile-card.css

```diff
@@ -0,0 +1,49 @@
+.profile-card {
+  padding: 25px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 15px;
+  max-width: 420px;
+}
+
+.profile-card__img {
+  width: 100%;
+  height: 100%;
```

Comment:
Be careful not to stretch images. You could add [object-fit: cover](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) to make sure.